main.py

```diff
@@ -49,33 +63,35 @@ def main():
     # Set up the prompt
     pre_prompt = (
         "<|begin_of_text|><|start_header_id|>system<|end_header_id|>\n"
-        "You are a git commit assistant. You are to take the given output of 'git diff --cached' and provide a succinct summary of the changes for a commit message. The only output you are to provide is that summary and it should be formatted in a multi-line output in only one of two ways:\n"
-        "1. If there are changes to a single file, the first line will be a summary of all changes, and each individual change will be summarized on its own line.\n"
-        "2. If there are changes to multiple files, the first line will be a summary of all changes, and each individual file will be summarized on its own line.\n"
+        "You are a git commit assistant. You are to take the given output of 'git diff --cached' and provide a succinct summary of the changes for a commit message. The only output you are to provide is that summary and it should be formatted in a multi-line output in only one of two ways:\n"  # noqa: E501
+        "1. If there are changes to a single file, the first line will be a summary of all changes, and each individual change will be summarized on its own line.\n"  # noqa: E501
+        "2. If there are changes to multiple files, the first line will be a summary of all changes, and each individual file will be summarized on its own line.\n"  # noqa: E501
         "Reminder: Do not output anything but one of the two options above.<|eot_id|>\n"
     )
 
     # Get the staged commit data
     staged_commit = "<|start_header_id|>user<|end_header_id|>\n"
-    staged_commit += subprocess.run(['git', 'diff', '--cached'], stdout=subprocess.PIPE, text=True).stdout
+    staged_commit += subprocess.run(
+        ["git", "diff", "--cached"], stdout=subprocess.PIPE, text=True
+    ).stdout
     staged_commit += "<|eot_id|>\n"
 
     # Combine everything into the full prompt
     post_prompt = "<|start_header_id|>assistant<|end_header_id|>"
     full_prompt = f"{pre_prompt}\n{staged_commit}\n{post_prompt}"
 
     # Run the model with the prompt
-    response = client.completions.create(
-      prompt=full_prompt,
-      temperature=0,
-      model=MODEL,
+    response = client.chat.completions.create(
+        messages=[{"role": "user", "content": full_prompt}],
+        model=MODEL,
     )
 
     # We are ready to provide the commit message
-    spinner.succeed('Done')
+    spinner.succeed("Done")
 
     # Commit with the generated message
-    subprocess.run(['git', 'commit', '-e', '-m', response.choices[0].text])
+    subprocess.run(["git", "commit", "-e", "-m", response.choices[0].message.content])
```

Comment:
The return from `chat/completions` is a bit different from `completions`, so this gets us the goods